Vagrantfile

```diff
@@ -165,6 +170,14 @@ def declare_bootstrap(machine, os_data)
     machine.vm.synced_folder ".", "/vagrant", type: "virtualbox"
   end
 
+  if os_data.has_key?(:scripts)
```

Comment:
Can be written as:

```ruby
os_data.fetch(:script, []).each do |script|
  machine.vm.provision script[:name],
    type: script[:type],
    inline: script[:data]
end
```

Reply:
I figured there'd be a ruby way of `.get(key, default)`. Thanks